src/lib/mdns/minimal/Server.cpp

```diff
@@ -61,6 +61,26 @@ class ShutdownOnError
     ServerBase * mServer;
 };
 
+chip::System::PacketBufferHandle Clone(chip::System::PacketBufferHandle & original)
```

Comment:
can you move this into the PacketBuffer class instead?

Reply:
I will move after I double-check if it is really needed. Had crashes before this, however I wonder if the crash was related to some packet buffer reference returns.

Reply:
Clone is needed. Moved it and added a comment that we waste one clone in the code and that could be resolved if we care enough (we may not, since mDNS is expensive as is and not as frequent since we use unicast for most purposes when clone is not needed)